File: 2016/gara1/permutazione/att/permutazione.pas
procedure permuta(K: longint; var P: array of char);
begin
    (* Mettete qui il codice della soluzione *)
    P[0] := 'A';
    P[1] := 'S';
    P[2] := 'D';
end;


var
    K: longint;
    P: array[0..25] of char;
    fr, fw: text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, K);

    permuta(K, P);

    writeln(fw, P);
    close(fr);
    close(fw);
end.

File: ed2/gara1/permutazione/att/permutazione.c
#include <stdio.h>
#include <assert.h>


void permuta(int K, char P[]) {
    // Mettete qui il codice della soluzione
    P[0] = 'A';
    P[1] = 'S';
    P[2] = 'D';
}


int main() {
    FILE *fr, *fw;
    int K;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(1 == fscanf(fr, "%d", &K));

    char P[27];
    permuta(K, P);
    P[26] = '\0';

    fprintf(fw, "%s\n", P);
    fclose(fr);
    fclose(fw);
    return 0;
}
